ncm-afsclt/src/test/resources/automatic.pan

```diff
@@ -0,0 +1,9 @@
+object template automatic;
+
```

Comment:
it would also be nice to do some schema tests, something similar like https://github.com/quattor/configuration-modules-core/blob/master/ncm-autofs/src/test/resources/maps.pan#L6 (with the lines above an below).
you will also have to replace https://github.com/quattor/configuration-modules-core/blob/master/ncm-afsclt/src/main/pan/components/afsclt/schema.pan#L8 with https://github.com/quattor/configuration-modules-core/blob/master/ncm-autofs/src/main/pan/components/autofs/schema.pan#L7